Scoop/Controller/Onboarding/OnboardingPageViewController.swift

```diff
@@ -93,6 +93,17 @@ extension OnboardingPageViewController: UIPageViewControllerDataSource {
         return index
     }
     
+    func setupNextButton(view: UIView, action: UIAction, button: UIButton) {
+        button.setImage(UIImage(named: "nextbutton"), for: .normal)
+        button.clipsToBounds = true
+        button.addAction(action, for: .touchUpInside) //EDIT LATER
+        view.addSubview(button)
+        
+        button.snp.makeConstraints { make in
+            make.leading.equalToSuperview().offset(282)
+            make.top.equalToSuperview().offset(601)
```

Comment:
This may look fine on certain devices, but did you test this on smaller iPhones (iPhone SE)? Or larger iPhones (pro max). It may not look how you want it to. A better solution would be to set these relative to `view.safeAreaLayoutGuide.bottom`